misc/package/Data Files/MWSE/core/mcm/components/settings/YesNoButton.lua

```diff
@@ -8,15 +8,16 @@
 --- The warnings arise because each field set here is also 'set' in the annotations in the core\meta\ folder.
 --- @diagnostic disable: duplicate-set-field
 
+
+
 local Parent = require("mcm.components.settings.Button")
 
+
 --- @class mwseMCMYesNoButton
 local YesNoButton = Parent:new()
-YesNoButton.defaultSetting = false
 
-function YesNoButton:getText()
-	local text = (self.variable.value and tes3.findGMST(tes3.gmst.sYes).value or tes3.findGMST(tes3.gmst.sNo).value)
-	return text --[[@as string]]
+function YesNoButton:convertToLabelValue(variableValue)
+	return tes3.findGMST(variableValue and tes3.gmst.sYes or tes3.gmst.sNo).value
 end
```

Comment:
I've just found out that the base Component class stores these two GMST values as `sYes` and `sNo`, so could make it just:
`return variableValue and self.sYes or self.sNo`